src/sas/qtgui/MainWindow/GuiManager.py

```diff
@@ -236,21 +233,6 @@ def closeAllPerspectives(self):
         self.loadedPerspectives = {}
         self._current_perspective = None
 
-    @staticmethod
```

Comment:
I thought this used to be handled by copying a `default_category.json`  file? Or was that pre-json in version 4 only? If the second then it was a bug introduced in transition but not necessarily part of the PR (though it would fit as "fixing" of category manager).

Reply:
That's how it is **currently** handled in main, but this PR removed that logic.

Reply:
my branch was missig `addCategories` in GuiManager. Added. This fixed the issue.